diff -up at-spi2-core-1.91.91/registryd/registry-main.c.crash-on-exit at-spi2-core-1.91.91/registryd/registry-main.c
--- at-spi2-core-1.91.91/registryd/registry-main.c.crash-on-exit	2011-03-09 19:05:00.774019630 -0500
+++ at-spi2-core-1.91.91/registryd/registry-main.c	2011-03-09 19:05:09.305912968 -0500
@@ -98,7 +98,7 @@ kill_accessibility_bus ()
   home = getenv ("HOME");
   if (!home)
     return;
-  name = g_strconcat (home, "/", ".atspi-dbus-bus.pid");
+  name = g_strconcat (home, "/", ".atspi-dbus-bus.pid", NULL);
   if (!name)
     return;
 
