diff -up at-spi2-core-1.91.93/registryd/registry-main.c.endsessionresponse at-spi2-core-1.91.93/registryd/registry-main.c
--- at-spi2-core-1.91.93/registryd/registry-main.c.endsessionresponse	2011-04-01 00:17:47.080690638 -0400
+++ at-spi2-core-1.91.93/registryd/registry-main.c	2011-04-01 00:17:16.184076898 -0400
@@ -107,7 +107,7 @@ end_session_response (gboolean is_okay, 
                 reason = "";
 
         ret = g_dbus_proxy_call_sync (client_proxy, "EndSessionResponse",
-                                      g_variant_new ("(us)", is_okay, reason),
+                                      g_variant_new ("(bs)", is_okay, reason),
                                       0, 1000, NULL, &error);
 
         if (!ret) {
